core/sail/spin/src/test/resources/functions/apf/concat.sparql

```diff
@@ -0,0 +1,5 @@
+prefix apf: <http://jena.hpl.hp.com/ARQ/property#>
```

Comment:
On 18 July 2016 at 23:10, pulquero <notifications@github.com> wrote:

> In core/sail/spin/src/test/resources/functions/apf/concat.sparql
> <https://github.com/eclipse/rdf4j/pull/240#discussion_r71240354>:
>
> > @@ -0,0 +1,5 @@
> > +prefix apf: <http://jena.hpl.hp.com/ARQ/property#>
>
> Conventionally file extension is .rq for sparql queries. I think it is
> mentioned in the sparql federation sparql if you are interested.
>
You are right with it. I changed. Sorry.


> Is it possible to have .srx files to check the results against? Check out
> rdf4j/testsuites/sparql/src/main/resources/testcases-sparql-1.1-w3c/functions
> to see how it can be done.
>
I will try to prepare. How are they compared with the results (I mean about
the pretty printing)?



> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse/rdf4j/pull/240/files/f73e4778ba5d7aca751f1713da178dbbd77d88f7#r71240354>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGbRXOo5Xw6jS5QdTavbu3q9eoU5RnZkks5qW_nvgaJpZM4JOCE0>
> .
>